xla/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -551,6 +556,12 @@ namespace wrap {
 // clang-format on
 #endif
 
+#ifdef MIOPEN_BETA_API
+#define MIOPEN_DNN_ROUTINE_EACH(__macro)                             \
+  __macro(miopenSetTensorDescriptorV2)                          \
```

Comment:
Also, I think this redefines the `MIOPEN_DNN_ROUTINE_EACH` macro. Did you mean to just add `__macro(miopenSetTensorDescriptorV2)` to the macro definition?